index.html

```diff
@@ -0,0 +1,396 @@
+                        <path d="M16 0C7.17706 0 0 7.17706 0 16C0 24.8229 7.17706 32 16 32C24.8229 32 32 24.8217 32 16C32 7.17831 24.8229 0 16 0ZM16 29.5213C8.54531 29.5213 2.47869 23.4559 2.47869 16C2.47869 8.54406 8.54531 2.47869 16 2.47869C23.4547 2.47869 29.5213 8.54406 29.5213 16C29.5213 23.4559 23.4559 29.5213 16 29.5213Z" fill="#6B6F7B"/>
+                        <path d="M22.1967 14.6491H17.2394V9.69174C17.2394 9.00761 16.6854 8.45236 16 8.45236C15.3146 8.45236 14.7606 9.00761 14.7606 9.69174V14.6491H9.80325C9.11787 14.6491 8.56387 15.2044 8.56387 15.8885C8.56387 16.5726 9.11787 17.1279 9.80325 17.1279H14.7606V22.0852C14.7606 22.7694 15.3146 23.3246 16 23.3246C16.6854 23.3246 17.2394 22.7694 17.2394 22.0852V17.1279H22.1967C22.8821 17.1279 23.4361 16.5726 23.4361 15.8885C23.4361 15.2044 22.8821 14.6491 22.1967 14.6491Z" fill="#6B6F7B"/>
+                    </svg>       
+                </button>
+            </div>
+        </section>
+        <div>
+            <h2>Transactions</h2>
+        </div>
+        <form>
+            <div>
+                <input type="text" placeholder="Search for....">
+                <img src="../img/search.png" alt="">
+                <input type="text" placeholder="Plug & Pay">
+                <img src="../img/Plug & Pay.png" alt="">
+                <input type="text" placeholder="Employee">
+                <img src="../img/user.png" alt="">
+                <input type="text" placeholder="Status">
+                <img src="../img/Status.png" alt="">
+                <input type="text" placeholder="Type">
+                <img src="../img/type.png" alt="">
+            </div>
+            <div>
+                <select name="" id="">
+                    <option value="">Sort by</option>
+                </select>
+            </div>
+        </form>
+        <button>
+            <img src="../img/download.png" alt="">
+        </button>
+
+        <table>
+            <tbody>
+                <tr>
+                    <th rowspan="2"></th>
+                    <th colspan="2" rowspan="2">Name</th>
+                    <th colspan="2" rowspan="2">User credentials</th>
+                    <th rowspan="2">Last Transcation</th>
+                    <th rowspan="2">Status</th>
+                    <th rowspan="2">End date </th>
+                    <th rowspan="2">Total Used</th>
+                    <th rowspan="2"></th>
+                </tr>
+                <tr>
+                    <th></th>
+                </tr>
+                <tr>
+                    <td rowspan="2">
+                        <input type="checkbox">
+                    </td>
+                    <td rowspan="2">
+                        <img src="../img/figma.png" alt="">
+                    </td>
+                    <td>
+                        <a href="https://www.figma.com/">Figma</a>
+                    </td>
+                    <td>
+                        <h3>Visa</h3>

```

Comment:
I wouldn't use h3 and h4 inside table cells since it's not a title but just piece of the content. You can use strong, em for example